wasm-syntax.md

```diff
@@ -34,14 +34,18 @@ WASM values are either integers or floating-point numbers, of 32- or 64-bit widt
 First, the type of values is built, then values annotated with their types are provided.
 
 ```k
+    syntax Data ::= Int | Float
```

Comment:
What is `Data`. A new type? Would it make sense to name it `Number`?

Reply:
`Data` is a new type added as a supertype of `Int` and `Float`. I think `Number` does sound better, will change.